lets-revise-hooks-today.md

```diff
@@ -0,0 +1,656 @@
+Hello Everyone
+
+I'm am Minhaj Ahmad Khan and I'm a Front-End developer. After a good response to my first blog. Thanks for your like and support so here I am back with a new blog I am sure you like it.
```

Comment:
I'm Minhaj Ahmad Khan and I'm a Front-End developer. After a good response to my first blog, I am back with a new blog and I hope you'll like it.

Reply:
Talk about what you'll cover in this article